Public/Src/Cache/ContentStore/Library/Stores/FileSystemContentStoreInternalChecker.cs

```diff
@@ -15,11 +14,69 @@
 using BuildXL.Cache.ContentStore.Tracing;
 using BuildXL.Cache.ContentStore.Tracing.Internal;
 using BuildXL.Cache.ContentStore.Utils;
+using BuildXL.Native.IO;
 using AbsolutePath = BuildXL.Cache.ContentStore.Interfaces.FileSystem.AbsolutePath;
 using FileInfo = BuildXL.Cache.ContentStore.Interfaces.FileSystem.FileInfo;
 
+#nullable enable
+
 namespace BuildXL.Cache.ContentStore.Stores
 {
+    /// <nodoc />
+    public sealed class SelfCheckSettings

```

Comment:
I think this is big enough to trigger the one class per file rule.

Reply:
Will move in a separate PR, because the current pipeline passes.